addons/website_slides/controllers/main.py

```diff
@@ -501,6 +534,10 @@ def channel(self, channel, category=None, tag=None, page=1, slide_category=None,
                 domain += [('slide_category', '=', slide_category)]
                 pager_url += "?slide_category=%s" % slide_category
 
+        if is_share_preview:
+            pager_args['invite_hash'] = kw.get('invite_hash')
+            pager_args['partner_id'] = kw.get('partner_id')
```

Comment:
if `is_share_preview`, we could simply use
```suggestion
            pager_args['invite_hash'] = invite_hash
            pager_args['partner_id'] = partner_id
```

But I think we could also add a `pagers_args` in `valid_invite_values` and instead use above `pager_args = valid_invite_values.get('pager_args', {})`.